Header header
StnArc[] arcs
string[] executedNodes